packages/palette/src/elements/index.tsx

```diff
@@ -24,6 +24,7 @@ export * from "./Flex"
 export * from "./FullBleed"
 export * from "./Grid"
 export * from "./GridColumns"
+export * from "./HorizontalOverflow"
```

Comment:
Should this be exposed in the docs or is this more of an internal component? We should be sure to add things there as new components are created, or at least stub out the file so things aren't buried unintentionally

Reply:
Ah, yeah I forgot; I extracted this for https://github.com/artsy/force/pull/8382 so it should be documented. Will follow up

Reply:
🙏